### ISSUE TITLE
"Expected {" error during formatting a certain expression

### ISSUE THREAD

**Issue by Cyberbeni** [nicklockwood/SwiftFormat#978]
v0.48.10
```swift
guard let parent = items.first?.parent, !items.contains(where: { $0.parent != parent }) else {
	return nil
}
```
```console
> swiftformat test.swift
Running SwiftFormat...
warning: No Swift version was specified, so some formatting features were disabled. Specify the version of Swift you are using with the --swiftversion option, or by adding a .swift-version file to your project.
error: Expected { on line 1 in /Users/benedek.kozma/Downloads/test.swift.
```

**Comment by CraigSiemens** [comment]
We just ran into this too. Same error on the second line.

```swift
if let trailingSwipeActions = self.tableView(tableView, trailingSwipeActionsConfigurationForRowAt: indexPath),
   trailingSwipeActions.actions.contains(where: { $0.style == .destructive }) {
    return .delete
}
```

**Comment by nicklockwood** [comment]
Sorry about that, fix incoming.

**Comment by nicklockwood** [comment]
@Cyberbeni, @CraigSiemens fixed in  0.48.11

**Comment by CraigSiemens** [comment]
Thanks!
